<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CacheBrowser</class>
 <widget class="QDialog" name="CacheBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WallpaperChanger</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>700</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="Label_Others">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Other Wallpapers&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Default">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Default Wallpaper&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Landscape">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Lanscape Wallpaper&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Portrait">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Portrait Wallpaper&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListView" name="ListView_Default">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="ListView_Landscape">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="ListView_Portrait">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="ListView_Others">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>620</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>20</width>
      <height>681</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Bnt_AddFromComputer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add New Wallpaper From Computer</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../res/green_plus.png</normaloff>../res/green_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Bnt_AddFromCache">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add New Wallpaper From Cache</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../res/orange_plus.png</normaloff>../res/orange_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Bnt_RemoveCache">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove A Wallpaper</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../res/red_minus.png</normaloff>../res/red_minus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Bnt_CacheInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wallpaper Information</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../res/blue_question.png</normaloff>../res/blue_question.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
